android/app/src/main/res/values/color.xml

```diff
@@ -22,4 +22,5 @@
     <color name="timestamp">#a0a3a7</color>
     <color name="link">#0a80d1</color>
     <color name="card_title">#ffffff</color>
+    <color name="card_title_background">#EF9A9A</color>
```

Comment:
This is a non themable, non standard color. Please align the card textview inside the card itself and remove the bg color